test/UserRepository/HtpasswdFactoryTest/ConfigImplementingArrayAccess.php

```diff
@@ -19,15 +19,11 @@
  */
 final class ConfigImplementingArrayAccess implements ArrayAccess
 {
-    /** @var array<array-key,mixed> */
-    private array $data;
-
     /**
-     * @param array<string,mixed> $data
+     * @param array<array-key,mixed> $data
      */
-    public function __construct(array $data)
+    public function __construct(private array $data)
     {
-        $this->data = $data;
     }
```

Comment:
What is it here? `array-key` or `string` ?

Reply:
on property, it defined as `array<array-key, mixed>`, on constructor param, it defined as `array<string, mixed>` so it inconsistent, I choose to use defined in property instead so psalm not show notice